2017/11.md

```diff
@@ -34,8 +34,12 @@ TBD Hotels nearby (optional)
 
 ## Agenda Topic Rules
 
-1. Proposals looking to advance must be added to the agenda along with necessary review materials 7 days prior to the first day of the meeting.
-   1. Note: any items added after 7 days prior to the start time of the meeting must be denoted with an ️:hourglass: symbol, which indicates that it is not eligible for stage advancement, and that it must wait until all other items have been addressed.
+Deadline for advancement eligibility: **November 18th, 2017, 18:00 UTC**
+
+1. Proposals not looking to advance may be added at any time; if after the deadline, please always use a pull request so that members are notified of changes.
+1. Proposals looking to advance to stage 0 or 1 must be added (and noted as such) prior to the deadline.
+1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline.
+    1. Necessary materials may include slides, a link to the proposal repo, a link to spec text, etc.
```

Comment:
Move this after the list of rules and change it to something like "Supporting materials includes slides, a link to the proposal repository, a link to spec text, etc.; essentially, anything you are planning to present to the committee, or that would be useful for delegates to review."